### DISCUSSION TITLE
No logging

### DISCUSSION THREAD

**Discussion by spmml** [kimai/kimai#2455] [Questions & Answers]
I have tested Kimai on 2 Devices. Now i do not get any logging messages in prod.log anymore. The file permissions are set to this:
-rwxrwxr-x 1 www-data www-data 169774 Mar 10 20:33 prod.log

I do not have a idea what is wrong.

**Comment by Scriptkiddi** [comment]
Check your webserver logs and clearing your cache probably does not hurt
